# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/FetchEventOpParent.h
# Commit: cf972fe0d961
# Full Hash: cf972fe0d96164788f998f04008930474526e400
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2022-02-26 21:36:36
# Regressor Bug: 1754365
# File Overlap Count: 1
# Description:
#   Bug 1754365 - Create IPC for transferring NavigationPreload's PerformanceTimingData. r=dom-worker-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138251
# ==============================================================================

diff -r f8afd25bf41e -r cf972fe0d961 dom/serviceworkers/FetchEventOpParent.h
--- a/dom/serviceworkers/FetchEventOpParent.h	Sat Feb 26 08:40:25 2022 +0000
+++ b/dom/serviceworkers/FetchEventOpParent.h	Sat Feb 26 08:40:26 2022 +0000
@@ -25,7 +25,7 @@
 
   // Transition from the Pending state to the Started state. Returns the preload
   // response, if it has already arrived.
-  Maybe<ParentToParentInternalResponse> OnStart(
+  Maybe<ParentToParentResponseWithTiming> OnStart(
       MovingNotNull<RefPtr<FetchEventOpProxyParent>> aFetchEventOpProxyParent);
 
   // Transition from the Started state to the Finished state.
@@ -37,12 +37,12 @@
   // IPDL methods
 
   mozilla::ipc::IPCResult RecvPreloadResponse(
-      ParentToParentInternalResponse&& aResponse);
+      ParentToParentResponseWithTiming&& aResponse);
 
   void ActorDestroy(ActorDestroyReason) override;
 
   struct Pending {
-    Maybe<ParentToParentInternalResponse> mPreloadResponse;
+    Maybe<ParentToParentResponseWithTiming> mPreloadResponse;
   };
 
   struct Started {